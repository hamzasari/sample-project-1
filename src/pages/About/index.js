import React from 'react';
import { Translate } from 'react-redux-i18n';

/**
 * About page
 * @returns {JSX.Element}
 */
const About = () => {
  return (
    <div className={`about-page-content-container`}>
      <h2 className={`title is-2`}>
        <Translate value={`headerLinks.about`} />
      </h2>
      <div className={`about-content-text`}>
        <Translate value={`pageContent.content1`} />
      </div>
      <div className={`about-content-text`}>
        <Translate value={`pageContent.content2`} />
      </div>
      <div className={`about-content-text`}>
        <Translate value={`pageContent.content3`} />
      </div>
    </div>
  );
};

export default About;
