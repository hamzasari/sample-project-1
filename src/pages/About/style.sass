.about-page-content-container
  padding: 20px 20px 0 20px

.about-content-text
  padding: 10px
