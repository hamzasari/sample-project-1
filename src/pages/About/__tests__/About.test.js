import React from 'react';
import renderer from 'react-test-renderer';
import About from '../index';

it('renders About snapshot', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
