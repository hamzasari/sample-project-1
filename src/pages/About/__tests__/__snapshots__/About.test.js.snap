// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders About snapshot 1`] = `
<div
  className="about-page-content-container"
>
  <h2
    className="title is-2"
  >
    <span>
      about
    </span>
  </h2>
  <div
    className="about-content-text"
  >
    <span>
      content1
    </span>
  </div>
  <div
    className="about-content-text"
  >
    <span>
      content2
    </span>
  </div>
  <div
    className="about-content-text"
  >
    <span>
      content3
    </span>
  </div>
</div>
`;
